tests/numerics/project_parallel_common.h

```diff
@@ -0,0 +1,358 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+// common framework to check whether an element of polynomial order p can
+// represent functions of order q
```

Comment:
Is this really what the test does? It seems about projection. Maybe expand the comment.